operations/iuf/workflows/preparation.md

```diff
@@ -35,6 +35,10 @@ Once this step has completed:
 
 - Environment variables have been set and required IUF directories have been created
 
+2. Ensure that the latest version of docs-csm is installed for the target CSM version being installed or upgraded.
+      
+    For example: when upgrading from CSM version 1.5.0 to version 1.5.1, install docs-csm-1.5.1.noarch
```

Comment:
Please add link to the procedure for checking for the latest documentation. https://github.com/Cray-HPE/docs-csm/blob/release/1.6/update_product_stream/README.md#check-for-latest-documentation